setup.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of DataCite.
+#
+# Copyright (C) 2015 CERN.
+#
+# DataCite is free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Setup module."""
```

Comment:
"""Python API wrapper for the DataCite Metadata Store API."""

and then later in `setup()` call we can use `description=__doc`.  (as @jirikuncar reminded me for intbitset)